src/data/Database.ts

```diff
@@ -26,7 +26,26 @@ class Database {
       migrationsRun: true,
       multipleStatements: true
     })
-    await createConnection(options)
+
+    try {
+      await createConnection(options)
+    } catch (err) {
+      console.error(err)
+      if (config.reconnect) {
+        console.info('Will reconnect to the DB after', config.reconnectDelay, 'ms later.')
+        this.reconnectWithDelay(config.reconnectDelay)
+      } else {
+        throw err
+      }
+    }
+  }
+
+  /**
+   * Reconnects to database after a delay.
+   */
+  async reconnectWithDelay (delay: number): Promise<void> {
+    await new Promise(resolve => setTimeout(resolve, delay))
```

Comment:
Can we remove this and use https://github.com/zeit/async-retry
Because we will need the retry functionality in other areas as well. It also has backoff feature (see the option for randomize)

Reply:
We can accept this as is and create a separate issue for that lib's implementation if you like